wallet-new/server/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -82,14 +86,11 @@ walletBackend WalletBackendParams {..} =
     -- Gets the Wai `Application` to run.
     getApplication :: SendActions WalletWebMode -> WalletWebMode Application
     getApplication sendActions = do
-      logInfo "DAEDALUS has STARTED!"
+      logInfo "Wallet Web API has STARTED!"
       saVar <- asks wwmcSendActions
       atomically $ STM.putTMVar saVar sendActions
-      when (isDebugMode walletRunMode) $ addInitialRichAccount 0
```

Comment:
@Martoon-00 Do we have a counterpart for this somewhere else? (i.e. I'm asking if that's fine to remove this line for good 😁 )

Reply:
Oh, exactly this like was once used to create wallet on startup for debugging, but it has not been maintained for a really long time, it can't be it still works :smile:

Reply:
I see, fine 😉